operator/controllers/syncer/csi_node.go

```diff
@@ -142,6 +145,26 @@ func (s *csiNodeSyncer) SyncCSIDaemonsetFn(daemonSetRestartedKey string, daemonS
 	out.Spec.Template.Spec.Tolerations = []corev1.Toleration{}
 	out.Spec.Template.Spec.ImagePullSecrets = []corev1.LocalObjectReference{}
 
+	// set daemonSetUpgradeStrategy
+	var maxUnavailableLocal intstr.IntOrString
+	if len(daemonMaxUnavailableGlobal) > 0 {
+		maxUnavailableLocal = intstr.FromString(daemonMaxUnavailableGlobal)
+	} else {
+		maxUnavailableLocal = intstr.FromInt(1)
+	}
+	logger.Info("UpdateStrategy for RollingUpdate set for ", "MaxUnavailable", maxUnavailableLocal)
+	deploy := appsv1.RollingUpdateDaemonSet{
+		MaxUnavailable: &maxUnavailableLocal,
+	}
+	var strategyType appsv1.DaemonSetUpdateStrategyType = "RollingUpdate"
```

Comment:
While doing above changes please also create a constant for "RollingUpdate" in constants.go and use it here